http/vertx/src/main/java/io/quarkus/ts/http/minimum/HelloResource.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.ts.http.minimum;
+
+import javax.inject.Inject;
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+
+import io.smallrye.mutiny.Uni;
+import io.vertx.mutiny.core.Vertx;
+import io.vertx.mutiny.core.buffer.Buffer;
+
+@Path("/hello")
+public class HelloResource {
+    private final Vertx vertx;
```

Comment:
Suggestion:

Also, you could inject Vertx by 

```
@Inject
    private Vertx vertx;
```

By this way the constructor is not required.

Reply:
All thing equal,I prefer to have explicit constructor and final fields.